browser/src/control/Control.RowHeader.ts

```diff
@@ -260,17 +263,20 @@ export class RowHeader extends cool.Header {
 
 	setOptimalHeightAuto(): void {
 		if (this._mouseOverEntry) {
-			const row = this._mouseOverEntry.index;
-			const command = {
-				Row: {
-					type: 'long',
-					value: row
-				},
-				Modifier: {
-					type: 'unsigned short',
-					value: 0
-				}
-			};
+			if (!this._hitResizeArea) {
+				const row = this._mouseOverEntry.index;
+				const command = {
+					Row: {
+						type: 'long',
+						value: row
+					},
+					Modifier: {
+						type: 'unsigned short',
+						value: 0
+					}
+				};
+				this._map.sendUnoCommand('.uno:SelectRow', command);
```

Comment:
I think we need a "return;" here, or it both resizes the column and selects it with double click.

Reply:
https://github.com/CollaboraOnline/online/pull/7910#pullrequestreview-1810977505

That's the same behavior here that @pedro approved.

> Because click event already sends the uno command. This seems to double the command.

The click is not active when `if (this._hitResizeArea)`, see the other line change.

Reply:
ping @gokaysatir

Reply:
> If you can check and do similar changes for also columns (if needed), that'd be great.

But what this isn't clear yet.

> Double click on row - sending the same command multiple times.

Is it ?
Do you mean Selecting the row then calling setOptimalHeight ?

As far I can tell this is necessary and I don't have a way around it, while functionally it does not hinder the user flow.